BUILD.md

```diff
@@ -39,3 +40,13 @@ Developers who can work a command line may be interested in using [Visual Studio
 - Run `cmake . -G "Ninja"` to generate build files.
 
 - Run `cmake --build .` to build the project.
+  
+## Linux
+### Steps
+1. Download the [Dockerfile.txt](https://github.com/R2Northstar/NorthstarLauncher/files/11956764/Dockerfile.txt) file. Rename it to just `Dockerfile` (GitHub wouldn't allow uploading otherwise).
```

Comment:
Why not directly adding the Dockerfile to the repository?
The one [mentioned here](https://github.com/R2Northstar/NorthstarLauncher/issues/501#issuecomment-1666499716) seems to be working at least on Ubuntu.

Reply:
Adding it is fine, any idea where I should add it to keep things cleaner? I'd imagine root directory is probably the best place for it (so alongside `BUILD.md`)

Reply:
Root dir is cool yeah

Reply:
done

Reply:
You should remove the .txt file extension + your first bullet point

Reply:
I guess this does work, I was just told github wouldnt let upload the file without the .txt when i was writing it